pom.xml

```diff
@@ -14,9 +14,9 @@
 
   <properties>
     <short.version>51</short.version>
-    <force.wsc.version>${version}</force.wsc.version>
-    <force.partner.api.version>${version}</force.partner.api.version>
-    <build.year>2020</build.year>
+    <force.wsc.version>${short.version}.0.0</force.wsc.version>
```

Comment:
How about we create a separate variable like `force.version` then set that to {short.version}.0.0 and then use `force.version` for `force.wsc.version` and `force.partner.api.version` That might be easier to read

Reply:
I am realizing that both version and short.version are used for versioning of artifacts, whereas force.wsc.version and force.partner.api.version are used for API version. So best to set these 2 pairs separately by explicitly setting force.wsc.version to a value which may/may not be the same as version.